hedera-node/hedera-app/src/main/java/com/hedera/node/app/uploader/credentials/BucketCredentials.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.uploader.credentials;
+
+import java.util.Objects;
+
+/**
+ * @param accessKey the access key of the bucket
+ * @param secretKey the secret key of the bucket
+ */
+public record BucketCredentials(String accessKey, String secretKey) {
```

Comment:
For the secret ley, a `char[]` is more secure because it can be zeroed, whereas a `string` retains the value in memory until GC occurs.

Reply:
Good call out. Have modified to use char array instead 💯